lib/nodes/decorators/While.ts

```diff
@@ -52,6 +52,7 @@ export class While<S extends BlueshellState, E> extends Decorator<S, E> {
 
 		if (res === rc.RUNNING) {
 			// yield to the behavior tree because the child node is running
+			storage.running = 0;
```

Comment:
Is this necessary? Decorator already does the same thing for you, so I think it's redundant